content/CORE/CORETutorials/SettingUIPreferences.md

```diff
@@ -20,9 +23,9 @@ This removes all the <i class="material-icons" aria-hidden="true" title="Visibil
 ## Creating a Custom Theme
 
 If one of the included themes doesn't satisfy your preference, a fully custom theme can be created.
-Click *CREATE NEW THEME* to start configuring the theme.
+Click **CREATE NEW THEME** to start configuring the theme.
 
-![Preferences Custom Theme](/images/CORE/12.0/PreferencesCustomTheme.png "Custom UI Theme")
+![Preferences Custom Theme](/images/CORE/13.0/PreferencesCustomTheme.png "Custom UI Theme")
 
```

Comment:
There are a few issues in lines 34 and 35:
Suggest changing line 34 from "toggle **Global Preview**" to "and click the **Global Preview** toggle."
line 35 suggest changing "topbar" to "top toolbar" so we are consistent in how we refer to the top toolbar in the documentation. Also I think we can change the second sentence to make it more present tense/active voice. How about this instead: "Color selection you make in the **Colors** tab determine the options you can select on the **General** tab." or something like this.

Reply:
Added "Color selection you make in the **COLORS** tab determine the options you can select on the **GENERAL** tab."